From: Giovanni Mascellani <gio@debian.org>
Subject: Fix .cabal file for using Debian bibutils package
Forwarded: not-needed
Last-Update: 2011-08-13

The .cabal files instructs the compiler to use the embedded bibutils
library. This patch makes it use the Debian packaged one.

Index: haskell-hs-bibutils-4.12/hs-bibutils.cabal
===================================================================
--- haskell-hs-bibutils-4.12.orig/hs-bibutils.cabal	2011-11-05 00:08:45.000000000 +0100
+++ haskell-hs-bibutils-4.12/hs-bibutils.cabal	2011-11-05 00:08:59.000000000 +0100
@@ -29,29 +29,6 @@
 maintainer:         andrea.rossato@unitn.it
 cabal-version:      >= 1.4
 build-type:         Simple
-extra-source-files:
-    bibutils/adsout.c bibutils/adsout.h bibutils/adsout_journals.c bibutils/bibcore.c bibutils/bibl.c
-    bibutils/bibl.h bibutils/biblatexin.c bibutils/biblatexin.h bibutils/bibtexin.c
-    bibutils/bibtexin.h bibutils/bibtexout.c bibutils/bibtexout.h bibutils/bibtextypes.c
-    bibutils/bibutils.c bibutils/bibutils.h bibutils/bltypes.c bibutils/charsets.c
-    bibutils/charsets.h bibutils/copacin.c bibutils/copacin.h bibutils/copactypes.c
-    bibutils/doi.c bibutils/doi.h bibutils/ebiin.c bibutils/ebiin.h
-    bibutils/endin.c bibutils/endin.h bibutils/endout.c bibutils/endout.h
-    bibutils/endtypes.c bibutils/endxmlin.c bibutils/endxmlin.h bibutils/entities.c
-    bibutils/entities.h bibutils/fields.c bibutils/fields.h bibutils/gb18030.c
-    bibutils/gb18030.h bibutils/gb18030_enumeration.c bibutils/is_ws.c bibutils/is_ws.h
-    bibutils/isiin.c bibutils/isiin.h bibutils/isiout.c bibutils/isiout.h
-    bibutils/isitypes.c bibutils/latex.c bibutils/latex.h bibutils/list.c
-    bibutils/list.h bibutils/marc.c bibutils/marc.h bibutils/medin.c
-    bibutils/medin.h bibutils/modsin.c bibutils/modsin.h bibutils/modsout.c
-    bibutils/modsout.h bibutils/modstypes.c bibutils/modstypes.h bibutils/name.c
-    bibutils/name.h bibutils/newstr.c bibutils/newstr.h bibutils/newstr_conv.c
-    bibutils/newstr_conv.h bibutils/reftypes.c bibutils/reftypes.h bibutils/risin.c
-    bibutils/risin.h bibutils/risout.c bibutils/risout.h bibutils/ristypes.c bibutils/rules.h
-    bibutils/serialno.c bibutils/serialno.h bibutils/strsearch.c bibutils/strsearch.h
-    bibutils/title.c bibutils/title.h bibutils/utf8.c bibutils/utf8.h bibutils/wordin.c
-    bibutils/wordin.h bibutils/wordout.c bibutils/wordout.h bibutils/xml.c
-    bibutils/xml.h bibutils/xml_encoding.c bibutils/xml_encoding.h
 
 library
     exposed-modules:  Text.Bibutils
@@ -60,22 +37,8 @@
     ghc-prof-options: -prof -auto-all
     hs-source-dirs:   src
     extensions:       ForeignFunctionInterface
-    include-dirs:     bibutils
-    includes:         bibutils.h
-    c-sources:
-        cbits/stub.c
-        bibutils/adsout.c bibutils/bibcore.c bibutils/bibl.c bibutils/biblatexin.c bibutils/bibtexin.c
-        bibutils/bibtexout.c bibutils/bibtextypes.c bibutils/bibutils.c bibutils/bltypes.c
-        bibutils/charsets.c bibutils/copacin.c bibutils/copactypes.c bibutils/doi.c
-        bibutils/ebiin.c bibutils/endin.c bibutils/endout.c bibutils/endtypes.c
-        bibutils/endxmlin.c bibutils/entities.c bibutils/fields.c bibutils/gb18030.c
-        bibutils/is_ws.c bibutils/isiin.c bibutils/isiout.c bibutils/isitypes.c
-        bibutils/latex.c bibutils/list.c bibutils/marc.c bibutils/medin.c
-        bibutils/modsin.c bibutils/modsout.c bibutils/modstypes.c bibutils/name.c
-        bibutils/newstr.c bibutils/newstr_conv.c bibutils/reftypes.c bibutils/risin.c
-        bibutils/risout.c bibutils/ristypes.c bibutils/serialno.c bibutils/strsearch.c
-        bibutils/title.c bibutils/utf8.c bibutils/wordin.c bibutils/wordout.c
-        bibutils/xml.c bibutils/xml_encoding.c
+    pkgconfig-depends: bibutils
+    c-sources:        cbits/stub.c
 
     if impl(ghc >= 6.10)
        build-depends: base >= 4, syb
